.gitignore

```diff
@@ -10,4 +10,4 @@ data.db
 npm-debug.log*
 .idea/
 .DS_Store/
-
+www/.DS_Store
```

Comment:
`.DS_Store` should cover everywhere. no?

Reply:
I thought so too. Apparently, it did not. 

I looked up and there's **/.DS_Store syntax, but need to check.